bin/seurat_merge.Rmd

```diff
@@ -128,12 +133,9 @@ VariableFeatures(combinedObj.integratedRNA) <- selectFeatures
 ```
 
 ```{r mergedObjDims, message=FALSE, echo=FALSE}
-# use variables to regress, if provided by user
-if (length(vars_to_regress_list) == 0 | vars_to_regress_list=="None") {
-  so_merged <- SCTransform(combinedObj.integratedRNA, assay = "RNA")
-} else {
-  so_merged <- SCTransform(combinedObj.integratedRNA, vars.to.regress = vars_to_regress_list, assay = "RNA")
-}
+# vars.to.regress is NULL by default
+so_merged <- SCTransform(combinedObj.integratedRNA, vars.to.regress = vars_to_regress_list, assay = "RNA")
```

Comment:
no need for an if/else here since we handled it at the start of the notebook, and `vars.to.regress` is `NULL` by default.